doc/semieunit.xml

```diff
@@ -0,0 +1,292 @@
+      of <Cite Key = "Howie1995aa"/>. The precise conditions
+      for a triple in &GAP; are as follows:
+
+      <List>
+        <Mark> M1 </Mark>
+        <Item>
+          <C>Y</C> is a semilattice which is a subdigraph of <C>X</C>.
+        </Item>
+        <Mark> M2 </Mark> 
+        <Item>
+          For all <C>A</C> in <C>X</C> and for all <C>B</C> in <C>Y</C>:
+          if <C>A</C> <M>\leq</M> <C>B</C> then <C>A</C> in <C>Y</C>.
+        </Item>
+        <Mark> M3 </Mark> 
+        <Item>
+          For every <C>A</C> in <C>X</C> there exists some <C>B</C> in
+          <C>Y</C> and <C>g</C> in <C>G</C> such that <C>A</C> is equal to
+          <C>B</C> acted on by <C>g</C>.
+        </Item>
+        <Mark> M4 </Mark> 
+        <Item>
+          For all <C>g</C> in <C>G</C> we have that the intersection of <C>Y</C>
+          acted on by <C>g</C> with <C>Y</C> is non-empty.
+        </Item>
+      </List>
+
+     <Example><![CDATA[
+gap> x := Digraph([[1], [1, 2], [1, 2, 3], [1, 4], [1, 4, 5]]);
+<digraph with 5 vertices, 11 edges>
+gap> y := InducedSubdigraph(x, [1, 2, 3]);
+<digraph with 3 vertices, 6 edges>
+gap> A := AutomorphismGroup(x);
+Group([ (2,4)(3,5) ])
+gap> S := McAlisterTripleSemigroup(A, OnPoints, x, y);
+<McAlister triple semigroup over Group([ (2,4)(3,5) ])>
+gap> T := McAlisterTripleSemigroup(A, x, y);
+<McAlister triple semigroup over Group([ (2,4)(3,5) ])>
+gap> T = S;
+true
+gap> GeneratorsOfSemigroup(T);
+[ (1, ()), (2, ()), (3, ()), (1, (2,4)(3,5)) ]
+gap> AsSemigroup(IsPartialPermSemigroup, T);
+<inverse partial perm monoid of size 4, rank 4 with 4 generators>]]></Example>
+    </Description>
+  </ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="McAlisterTripleSemigroup">
+  <ManSection>
+    <Oper Name = "McAlisterTripleSemigroup" Arg = "G, act, X, Y"
+    Label = "for a group, action, partial order digraph, and semilattice digraph"/>
+    <Oper Name = "McAlisterTripleSemigroup" Arg = "G, X, Y"
+    Label = "for a group, partial order digraph, and semilattice digraph"/>
+    <Oper Name = "McAlisterTripleSemigroup" Arg = "G, act, X, sub_ver"
+    Label = "for a group, action, partial order digraph, and homogeneous list"/>
+    <Oper Name = "McAlisterTripleSemigroup" Arg = "G, X, sub_ver"
+    Label = "for a group, partial order digraph, and homogeneous list"/>
+    <Returns>A McAlister triple semigroup.</Returns>
+    <Description>
+      <C>McAlisterTripleSemigroup</C> requires a digraph <A>X</A> satisfying
+      <Ref Oper= "IsPartialOrderDigraph" BookName= "Digraphs" />, a 
+      <Ref Oper= "Digraph" BookName= "Digraphs" /> <A>Y</A> satisfying
+      <Ref Oper= "IsJoinSemilatticeDigraph" BookName= "Digraphs" />, a group
+      <A>G</A>, and a function <A>act</A> defining an action of <A>G</A> on
+      <A>X</A>. <C>McAlisterTripleSemigroup</C> also has a three argument
+      version which assumes that  <A>act</A> is <Ref Func= "OnPoints"
+      BookName= "GAPDoc"/>. Furthermore there are three and four argument
```

Comment:
Shouldn't be `GAPDoc` here. Probably should be `ref`. Change it to ref and see if the warnings go away when you compile the documentation.